addons/website_slides/static/src/scss/website_slides.scss

```diff
@@ -530,7 +533,11 @@ $line-height-truncate: 1.25em;
                 top: -58px;
             }
         }
-
+        .o_wslides_lesson_side_card{
```

Comment:
I think we're missing blank lines before and after this block, to match the rest of the file?